### DISCUSSION TITLE
Progress Report 23–27 May

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#24] [Progress Report]
🚴🏻 **Progress**
None. It was a Science On a Sphere heavy week.

😖 **Regressions**
None.

💬 **To Discuss**
- Contents of the NetCDF diag files

📆 **Next Week**
- Vacation. I’ll be back on 6 June

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**
- VLab has set up the cluster and is exploring how we can deploy containers into it while meeting their security requirements
- Set up CI for the frontend and the backend
  - #21
  - #22
- It became apparent it would be useful to be able to manually trigger CI to produce our docker images - that change is in review (#23)

😖 **Regressions**
- None.

💬 **To Discuss**
- Contents of the NetCDF diag files

📆 **Next Week**
- Continue working with VLab on Kubernetes
- Explore reading from the NetCDF diag files
